android/app/src/main/java/org/fossasia/openevent/api/Urls.java

```diff
@@ -60,4 +61,14 @@ public static void setBaseUrl(String baseUrl) {
 
         }
     }
+
+    /**
+     * Checks if the app link is replaced by the generator, if not
+     * returns null which is to be checked by caller to make decision
+     * accordingly.
+     * @return String
+     */
+    public static String getAppLink() {
+        return APP_LINK.equals("https://app_link_goes_here.com")?null:APP_LINK;
```

Comment:
No need to return null, we can return the play store homepage instead, that's a better option imo.